zap/src/output/luau/client.rs

```diff
@@ -174,6 +174,23 @@ impl<'src> ClientOutput<'src> {
 		));
 	}
 
+	fn get_values(&self, data: &Option<Vec<Ty>>) -> String {
+		if let Some(types) = data {
+			(1..=types.len())
+				.map(|i| {
+					if i == 1 {
+						"value".to_string()
+					} else {
+						format!("value{}", i)
+					}
+				})
```

Comment:
If you apply my comment above:
```suggestion
				.map(|i| format!("value{}", i))
```